builtin/tag.c

```diff
@@ -106,7 +106,22 @@ static int delete_tag(const char *name, const char *ref,
 static int verify_tag(const char *name, const char *ref,
 				const unsigned char *sha1)
```

Comment:
Yep, I think we need to do this on the commit that adds the --format flag. I'll check the docs also (maybe those can be separate commits)